processing/src/main/java/io/druid/query/lookup/LookupModule.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.lookup;
+
+
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Binder;
+import io.druid.guice.JsonConfigProvider;
+import io.druid.guice.LifecycleModule;
+import io.druid.initialization.DruidModule;
+
+import java.util.List;
+import java.util.UUID;
+
+//TODO unify this with PR/1576
+public class LookupModule implements DruidModule
+{
+  private static final String PROPERTY_BASE = "druid.lookup";
+
+  @Override
+  public List<? extends Module> getJacksonModules()
+  {
+    return ImmutableList.of(new SimpleModule("LookupModule-" + UUID.randomUUID().toString()));
```

Comment:
To be honest i never understood the diff, can you explain when we need module name ?